app/jobs/claims_export_job.rb

```diff
@@ -1,20 +0,0 @@
-# frozen_string_literal: true
```

Comment:
This file has moved into the atos export gem - I missed it in the last change.  It is purely for ATOS, so belongs in the gem